SEED-platform/seed
# Deactivate orphaned users when removed from last org

#### What's this PR do?
If a user only belongs to one org and is removed from that org (either from the admin page or the org member page) the user will be prompted to confirm that that will delete the user (rather than orphaning them).

Also:
- Fixes add/remove user form validation and console error
- Fixes a variety of small bugs on the admin page
- Fixes a bug with updating the default org for a user that was removed

#### How should this be manually tested?
1. Add a user to at least two orgs
2. Delete the user from those orgs and confirm that the last one prompts you to delete the user
#### What are the relevant tickets?
#994 
#### Screenshots (if appropriate)
![2019-12-16_220919](https://user-images.githubusercontent.com/411466/70967047-58f3c800-2052-11ea-87b8-d0dc065dd005.png)
